internal/pkg/manifest/validate_test.go

```diff
@@ -1727,13 +1727,13 @@ func TestPlacement_Validate(t *testing.T) {
 	}
 }
 
-func TestRdwsPlacement_Validate(t *testing.T) {
+func TestRequestDrivenWebServicePlacement_Validate(t *testing.T) {
 	testCases := map[string]struct {
-		in     *rdwsPlacement
+		in     *RequestDrivenWebServicePlacement
 		wanted error
 	}{
 		"should return an error if placement is public": {
-			in:     (*rdwsPlacement)(aws.String("public")),
+			in:     (*RequestDrivenWebServicePlacement)(aws.String("public")),
 			wanted: errors.New(`placement "public" is not supported for Request-Driven Web Service`),
```

Comment:
Q: (I wonder if the error message was introduced in a different PR? Sorry if I missed it)...Should we explain a little bit more here so users know what the options are and what they mean? Or should we just let them refer to the docs? 🤔

Reply:
The doc hasn't been out yet. I think they should refer to the doc otherwise they won't know how to set this field as we don't write it to the manifest by default.

Reply:
I think it also makes sense to say it in the initial manifest that Copilot produces